src/main/kotlin/org/malachite/estella/services/ApplicationService.kt

```diff
@@ -128,4 +138,10 @@ class ApplicationService(
         }
     }
 
+    data class ApplicationWithStagesAndOfferName(
+        val application: Application,
+        val stages: List<RecruitmentStage>,
+        val offerName: String
+    )
+
```

Comment:
To jest ta klasa co się u mnie w review miałeś uwagę??

Reply:
Istnieje spora szansa, że tak XDDD